### ISSUE TITLE
Interpolation inside an interpolation isn't handled correctly

### ISSUE THREAD

**Issue by xzyfer** [sass/libsass#615]
The following results in an `invalid property name` error.

```scss
$foo: "bar";
%#{"foo--#{$foo}"} {
    foo: bar;
}

a { 
  @extend %foo--bar; 
}
```

I suspect this is partly responsible for https://github.com/sass/libsass/issues/439 as well.

Specs added https://github.com/sass/sass-spec/pull/123.

**Comment by jmhe** [comment]
Created a pull request for this:

https://github.com/sass/libsass/pull/854

**Comment by mgreter** [comment]
This should be solved by #910!
